crates/proof-of-sql/src/sql/proof_gadgets/range_check.rs

```diff
@@ -91,6 +117,82 @@ pub fn final_round_evaluate_range_check<'a, S: Scalar + 'a>(
     );
 }
 
+/// Verify that the prover claim is correct.
+///
+/// # Panics
+///
+/// if a column contains values outside of the selected range.
+pub fn verifier_evaluate_range_check<C>(
```

Comment:
Let's use `S` for `Scalar` to be aligned with the rest of the project. `C` normally refers to commitments.